core/events.js

```diff
@@ -1086,6 +1095,43 @@ Blockly.Events.VarRename.prototype.run = function(forward) {
 };
 
 /**
+ * Class for a variable disable event.
+ * @param {Blockly.VariableModel} variable The disabled variable.
+ *     Null for a blank event.
+ * @extends {Blockly.Events.Abstract}
+ * @constructor
+ */
+Blockly.Events.VarDisable = function(variable) {
+  if (!variable) {
+    return;  // Blank event to be populated by fromJson.
+  }
+  Blockly.Events.VarDisable.superClass_.constructor.call(this, variable);
+};
+goog.inherits(Blockly.Events.VarDisable, Blockly.Events.Abstract);
+
+/**
+ * Type of this event.
+ * @type {string}
+ */
+Blockly.Events.VarDisable.prototype.type = Blockly.Events.VAR_DISABLE;
+
+/**
+ * Run a variable disable event.
+ * @param {boolean} forward True if run forward, false if run backward (undo).
+ */
+Blockly.Events.VarDisable.prototype.run = function(forward) {
+  var workspace = this.getEventWorkspace_();
+  if (!workspace) return;
```

Comment:
Put the return on a new line and add braces around it.

The style guide technically says this is okay, but it's not the style that the rest of Blockly uses.